package.json

```diff
@@ -159,6 +159,7 @@
     "react-native-paper": "^5.3.1",
     "react-native-vector-icons": "^9.2.0",
     "react-navigation-header-buttons": "^9.0.1",
-    "react-use": "^17.4.0"
+    "react-use": "^17.4.0",
+    "zustand": "^4.4.7"
```

Comment:
maybe my only concern is adding this as a dep to the sdk

Reply:
@jkdowdle Can you elaborate on the concern?